bin/proxy.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-console */
+require('babel-register');
+
+const http = require('http');
+
+const config = require('config');
+const httpProxy = require('http-proxy');
+
+
+const proxy = httpProxy.createProxyServer({ secure: false });
+const apiHost = config.get('proxyApiHost', null) || config.get('apiHost');
+const frontendHost = `http://${config.get('serverHost')}:${config.get('serverPort')}`;
+
+console.log(`apiHost: ${apiHost}`);
+console.log(`frontendHost: ${frontendHost}`);
+
+function getHost(req) {
+  if (!req.headers['user-agent'].includes('Android') || req.url.startsWith('/api/')) {
+    return apiHost;
+  }
+  return frontendHost;
+}
+
+const server = http.createServer((req, res) => {
+  console.log(`${req.method} ${req.url}`);
+  return proxy.web(req, res, { target: getHost(req), changeOrigin: true, autoRewrite: true });
+});
+
+proxy.on('proxyRes', (proxyRes, req) => {
+  console.log(` ${proxyRes.statusCode} ~> ${proxyRes.client._host}${req.url}`);
```

Comment:
Similarly, I don't think this is so helpful either. Maybe the logging could only happen for 4xx and 5xx responses?